services/sharepoint-connector/src/unique-api/unique-files/unique-files.consts.ts

```diff
@@ -0,0 +1,72 @@
+import { gql } from 'graphql-request';
+import type { UniqueFile } from './unique-files.types';
+
+export interface ContentUpdateMutationInput {
+  contentId: string;
+  ownerId?: string;
+  input: Record<string, never>;
+}
+
+export interface ContentUpdateMutationResult {
+  contentUpdate: {
+    id: string;
+    ownerId: string;
+    ownerType: string;
+  };
+}
+
+export const CONTENT_UPDATE_MUTATION = gql`
+  mutation ContentUpdate($contentId: String!, $ownerId: String, $input: ContentUpdateInput!) {
+    contentUpdate(contentId: $contentId, ownerId: $ownerId, input: $input) {
+      id
+      ownerId
+      ownerType
+    }
+  }
+`;
+
+export interface ContentDeleteMutationInput {
+  contentDeleteId: string;
+}
+
+export interface ContentDeleteMutationResult {
+  contentDelete: boolean;
+}
+
+export const CONTENT_DELETE_MUTATION = gql`
+  mutation ContentDelete($contentDeleteId: String!) {
+    contentDelete(id: $contentDeleteId)
+  }
+`;
+
+export interface PaginatedContentQueryInput {
+  skip: number;
+  take: number;
+  where: {
+    key?: {
+      startsWith?: string;
```

Comment:
i'll need to send the actual keys in but this implementation details I can change on local development